src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century ? people
+    .filter(person => person.sex === 'm' && Math
+      .ceil(person.died / 100) === century) : people
+    .filter(person => person.sex === 'm');
+
+  return men
+    .reduce((prev, cur) => prev + (cur.died - cur.born), 0) / men.length;
```

Comment:
You do the same on calculateWomenAverageAge. You can create a separate function and use it here and on calculateWomenAverageAge